src/webfaf/manage.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
```

Comment:
Change this please to `#!/usr/bin/env python`  so it can be run inside virtualenv like ./manage.py